FastAutoAugment/data.py

```diff
@@ -212,14 +213,14 @@ def get_dataloaders(dataset, batch, dataroot, split=0.15, split_idx=0, multinode
             logger.info(f'----- dataset with DistributedSampler  {dist.get_rank()}/{dist.get_world_size()}')
 
     trainloader = torch.utils.data.DataLoader(
-        total_trainset, batch_size=batch, shuffle=True if train_sampler is None else False, num_workers=8, pin_memory=True,
-        sampler=train_sampler, drop_last=True)
+        total_trainset, batch_size=batch, shuffle=True if train_sampler is None else False, num_workers=1, pin_memory=True,
+        sampler=train_sampler, drop_last=False)
     validloader = torch.utils.data.DataLoader(
-        total_trainset, batch_size=batch, shuffle=False, num_workers=4, pin_memory=True,
+        total_trainset, batch_size=batch, shuffle=False, num_workers=1, pin_memory=True,
         sampler=valid_sampler, drop_last=False)
 
     testloader = torch.utils.data.DataLoader(
-        testset, batch_size=batch, shuffle=False, num_workers=8, pin_memory=True,
+        testset, batch_size=batch, shuffle=False, num_workers=1, pin_memory=True,
```

Comment:
why change the num_workers? Also, make this a parameter - that is why the code was slow - your g4.4x has 16 workers - if you use only 4 - it will be very slow.